vendor/engines/saml_authentication/lib/saml_authentication/idp_entity_id_reader.rb

```diff
@@ -0,0 +1,40 @@
+module SamlAuthentication
+
+  # The default IdpEntityIdReader triggers an error on SLO because that request to
```

Comment:
I'm not entirely following what the error is that we're trying to handle here--might be good to walk through it together.